lib/main.dart

```diff
@@ -286,6 +289,7 @@ class TaskList extends StatelessWidget {
     final tasks = isPending ? appState.pendingTasks : appState.completedTasks;
 
     // Building the task list
+    //Majd
```

Comment:
consider adding a check for empty task lists and display a user-friendly message when no tasks are available.